src/makeCalculator.js

```diff
@@ -36,8 +36,48 @@
  *
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
+  const ops = {
+    'result': 0,
+    'add': makeAdd,
+    'subtract': makeSubst,
+    'multiply': makeMult,
+    'divide': makeDiv,
+    'reset': makeReset,
+    'operate': makeOperate,
+
+  };
+
+  function makeOperate(operation, value) {
+    ops.result = operation(ops.result, value);
+
+    return ops;
+  }
+
+  function makeAdd(a, b) {
```

Comment:
write it (all functions) as methods and use `this.result` instead of passing the second argument

Reply:
Please give me a hint, I didn't get it, what I suppose to assign to this? 
When I set ops to this it throws me an error

Reply:
if you have any questions, just feel free to ask in `fe_chat`